glossary.yml

```diff
@@ -2962,6 +2962,11 @@
     term: ቃል
     def: >
       እንደ ግስ፣ በዳታቤዝ ወይም በ[ቨርዥን መቆጣጠሪያ](#version_control_system) ላይ የተወሰኑ ለውጦችን የማስቀመጥ ተግባር [መዝገበ](#repository) ::እነዚህ ለውጦች እንደ ስም ተቀምጠዋል ።
+  uk:
+    term: "коміт"
+    def: >
+      Робити коміт - зберегти набір змін до бази даних або [репозиторій](#repository) [системи контролю версій](#version_control_system). 
```

Comment:
We want to convey that comit can be used as a verb and as a noun though...

Reply:
I think we had this conversation with the "fork" term #706: 
 - "Я не впевнений що в Українській мові ми можемо сказати "форк" як дієслово. "Зробити fork" буде означати зробити гілку, але тут дієслово вже "зробити" а не "fork". Ми також можемо сказати "фократи", але це вже сленг."
- "Саме так! Взагалі, я мав на увазі не використовувати "форк" як дієслово, а пояснити, що у англійській мові воно може бути так використано. ОК, залишимо як є, тоді тільки змінить "власник якої"
We can't exactly "convey" this term to make it noun unless we use slang "коммітити". We can explain that it could be used as a verb in English, but, I think, that we should translate it only as a noun.

Reply:
I see your point. But in this case commit as "Збереження набору змін..." is a process. Still does not look optimal...

Reply:
We could have 2 paragraphs.
"Збереження набору змін..."
"And a noun definition..."
It would make it work I think.